lib/flushbar.dart

```diff
@@ -8,50 +8,50 @@ import 'flushbar_route.dart' as route;
 
 const String FLUSHBAR_ROUTE_NAME = "/flushbarRoute";
 
-typedef void FlushbarStatusCallback(FlushbarStatus status);
+typedef void FlushbarStatusCallback(FlushbarStatus? status);
```

Comment:
Shouldn't callback be called always with some status? I don't think it can be null.